examples/tutorials/hnn_tutorial.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>Please start with a brief introduction about deepchem and HNN model. HNNs are well explained, but talk about deepchem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>